### PR TITLE
Additional clarifications on power flow inputs (SSH) and state values (SV)

### PR DESCRIPTION
**Pull request by marqueslanauja**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

The current configuration parameter `profileUsedForInitialFlowsValues `has been changed to `readShuntSectionsTapPositionsFromSVIfPresent`. By default is set to false.
Now this parameter only decides the source (SSH or SV) for shunt sections and tap positions.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java [hunk 1]
```diff
@@ -176,11 +176,11 @@ private String tripleStore(Properties p) {
                                 p,
                                 CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE_PARAMETER,
                                 defaultValueConfig))
-                .setProfileUsedForInitialStateValues(
-                        ConversionParameters.readStringParameter(
+                .setReadShuntSectionsTapPositionsFromSVIfPresen(
```
Comment (zamarrenolm) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java:179]: typo: should be `...SVIfpresent`

**Review by zamarrenolm — CHANGES_REQUESTED** [review]

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java (hunk 1 above)
Comment (marqueslanauja) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java [hunk 2]
```diff
@@ -306,11 +306,11 @@ private void copyStream(ReadOnlyDataSource from, DataSource to, String fromName,
             "The triplestore used during the import",
             TripleStoreFactory.defaultImplementation())
             .addAdditionalNames("powsyblTripleStore");
-    private static final Parameter PROFILE_USED_FOR_INITIAL_STATE_VALUES_PARAMETER = new Parameter(
-            PROFILE_USED_FOR_INITIAL_STATE_VALUES,
-            ParameterType.STRING,
-            "Profile used for initial state values",
-            "SSH");
+    private static final Parameter READ_SHUNT_SECTIONS_TAP_POSITIONS_FROM_SV_IF_PRESENT_PARAMETER = new Parameter(
```
Comment (miovd) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java:309]: Can you keep the previous name as additional name for compatibility?

**Review by miovd — APPROVED** [review]

**Comment by miovd** [comment]
@zamarrenolm If it is okay for you, we can merge this PR

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java (hunk 2 above)
Comment (marqueslanauja) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java]: Done. I made additional modifications to keep the type of the parameter to allow the previous parameter for compatibility.

**Review by zamarrenolm — APPROVED** [review]

**Comment by zamarrenolm** [comment]
> @zamarrenolm If it is okay for you, we can merge this PR

just approved, it is ok for me

**Pull request merged**